### PR TITLE
Step package framework blog post

### PR DESCRIPTION
**Pull request by robpearson**
# About

This is a post about the step package framework.

# Release

This is related to the Octopus 2022 Q2 launch and we'll add a link to this blog post afterwards.

# Blog image idea

Blog image is done.

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [writing for Octopus TL;DR](https://www.octopus.design/writing/writing-tldr/).
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [x] The images I've included follow the [rules for working with images](https://www.octopus.design/writing/working-with-images/).
- [x] I've added alt text for all the images I've included (alt text describes image to people unable to see it, 125 characters max)
- [x] One screenshot (if any are included) is suitable for Twitter. This screenshot should be 16:9 ratio and at least 1200 x 675 pixels. For screenshots from the Octopus Web Portal, please use Chrome and the browser’s zoom function to remove as much white space as possible. If screenshots are annotated (e.g., with boxes or arrows), please also provide a version that is unannotated for use on Twitter. Do not use dark mode for the Octopus Web Portal.

## How to review this PR

N/A

## Publication date

8-August-2022. Tegan is already aware of this one.

### CODE REVIEW

File: blog/2022-08/step-package-framework/index.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+## Conclusion
+
+The step package framework is changing how we build and deliver new steps to your Octopus Server instances. It means that new steps can be delivered faster, and without the need for upgrades. 
+
+We look forward to bringing you new and improved steps using this framework as we roll out even more functionality. 
+
+Happy deployments!
+
+*Thanks to Shaun Hevey, Matt Shepherd, Ray Nham, and Igor Pavlikhin who all contributed to this blog post.*
```
Comment (TeganAli) [inline blog/2022-08/step-package-framework/index.md:240]:
```suggestion
*Thanks to Shaun Hevey, Matt Shepherd, Ray Nham, and Egor Pavlikhin who all contributed to this blog post.*
```

File: blog/2022-08/step-package-framework/index.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+## Conclusion
+
+The step package framework is changing how we build and deliver new steps to your Octopus Server instances. It means that new steps can be delivered faster, and without the need for upgrades. 
+
+We look forward to bringing you new and improved steps using this framework as we roll out even more functionality. 
+
+Happy deployments!
+
+*Thanks to Shaun Hevey, Matt Shepherd, Ray Nham, and Egor Pavlikhin who all contributed to this blog post.*
```
Comment (mattocto) [inline blog/2022-08/step-package-framework/index.md:240]: Please can you put my name last? I didn't really write any of it, I just helped Shaun with some of the editing.

As Shaun is listed as the post author, I'm not sure his name needs to be in this section at all?

File: blog/2022-08/step-package-framework/index.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
+---
+title: Improving delivery of your deployment steps
+description: Our new step package framework means we can deliver new steps can to your Octopus instance faster, and without the need for upgrades. Learn more.
```
Comment (mattocto) [inline blog/2022-08/step-package-framework/index.md:3]: > means we can deliver new steps **can** to your Octopus instance faster

There's an extra "can" in this sentence.

**Review by mattocto — APPROVED** [review]
I made a couple of minor changes, but the rest looks good to me.

File: blog/2022-08/step-package-framework/index.md (hunk 3 above)
Comment (TeganAli) [inline blog/2022-08/step-package-framework/index.md]:
```suggestion
description: Our new step package framework means we can deliver new steps to your Octopus instance faster, and without the need for upgrades. Learn more.
```

**Review by shaun-od — APPROVED** [review]
LGTM

**Pull request merged**